### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by altro3**

### CODE REVIEW

File: config/checkstyle/suppressions.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 <suppressions>
     <!-- <suppress checks="FileLength" -->
-              <!-- files="DefaultBeanContext.java|BeanDefinitionWriter.java|DefaultHttpClient.java"/> -->
+    <!-- files="DefaultBeanContext.java|BeanDefinitionWriter.java|DefaultHttpClient.java"/> -->
```
Comment (dstepanov) [inline config/checkstyle/suppressions.xml]: I belive this file is always overridden from the template

**Review by dstepanov — APPROVED** [review]

File: config/checkstyle/suppressions.xml (hunk 1 above)
Comment (altro3) [reply config/checkstyle/suppressions.xml]: Agree. Fixed :-)

**Comment by altro3** [comment]
@dstepanov You can close this issue: https://github.com/micronaut-projects/micronaut-jaxrs/issues/450 .You already fixed it

**Comment by dstepanov** [comment]
I have created and changed the pr to 4.7.x

**Comment by dstepanov** [comment]
Thanks!

**Comment by altro3** [comment]
Need to set `4.7.x` as default branch. And merge `4.6.x` to it

**Comment by dstepanov** [comment]
It should be merged

**Comment by altro3** [comment]
yeah, it's true. Need tos set branch as default

**Pull request merged**